scripts/generate_error_code.hs

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env stack
+-- stack --resolver lts-17.3 script --package string-interpolate --package universum --package text --package text-manipulate
+
+-- SPDX-FileCopyrightText: 2021 Tocqueville Group
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+-- | Script for adding new error to ligo and also update the docs.
+module GenerateErrorCode where
+
+import Data.String.Interpolate (i)
+import qualified Data.Text as T
+import Data.Text.Manipulate (toCamel)
+import Prelude ()
+import Universum
+
+data ErrorItem = ErrorItem
+  { eiLabel :: Text
+  , eiCode :: Integer
+  , eiDesc :: Text
+  }
+
+data ShortErrorItem = Text :? Text
+  deriving (Eq)
+
+removedError :: ShortErrorItem
+removedError = "" :? ""
+
+errorsEnumerate :: Integer -> [ShortErrorItem] -> [ErrorItem]
+errorsEnumerate start =
+  map (\(eiCode, eiLabel :? eiDesc) -> ErrorItem{..}) .
+  filter ((/= removedError) . snd) .
+  zip [start..]
+
+invalidInputErrors :: [ErrorItem]
+invalidInputErrors = errorsEnumerate 100
+  -- New errors must be added to the _end_ of this list.
+  -- To remove an error, replace it with 'removedError'.
+
+  [ "not_admin"
+    :? "The sender is not the administrator"
+  , "not_pending_admin"
+    :? "Authorized sender is not the current pending administrator"
+  , "not_token_owner"
+    :? "Trying to configure operators for a different wallet which sender does not own"
```

Comment:
It looks like this is unused and so should be removed.

Reply:
Removed.